tests/atoms/gaia_data_layer.js

```diff
@@ -45,16 +45,17 @@ var GaiaDataLayer = {
 
   disableBluetooth: function() {
     var bluetooth = window.navigator.mozBluetooth;
-    if (bluetooth.enabled) {
+    if (bluetooth.defaultAdapter.state == 'enabled') {
```

Comment:
I don't think so. `disabling` also exist and I don't know what could happen if we ask to disable when we're disabling.